### PR TITLE
Update SFX for mod select overlay show/hide

### PR DESCRIPTION
**Pull request by nekodex**
- [x] depends on ppy/osu-resources#202

### CODE REVIEW

**Comment by peppy** [comment]
Two things to note with this (non-critical but maybe worth considering?):
- Because the initial display of the mod screen favours the columns being further to the right, the balance of the individual columns feels lob-sided towards the right. It's probably best to fake it and offset to allow them to feel balanced (ie. add a left bias equal to the visual offset of the first column).
- The sound of the clicking of the "mods" button at song select is a bit overpowering. I don't think it really fits that well, and would probably work better as one of the softer button click sounds (especially since all three of the buttons there have their own sounds that play after click now).

**Comment by nekodex** [comment]
> * Because the initial display of the mod screen favours the columns being further to the right, the balance of the individual columns feels lob-sided towards the right. It's probably best to fake it and offset to allow them to feel balanced (ie. add a left bias equal to the visual offset of the first column).

Yeah, that makes sense. I guess the only thing I'd check is whether it feels off when the screen aspect ratio is wide enough such that the columns appear completely on the left.

> * The sound of the clicking of the "mods" button at song select is a bit overpowering. I don't think it really fits that well, and would probably work better as one of the softer button click sounds (especially since all three of the buttons there have their own sounds that play after click now).

I still consider those mod button sounds as placeholders, so feel free to swap it for something else for the time being.

File: osu.Game/Overlays/Mods/ModSelectOverlay.cs [hunk 1]
```diff
@@ -453,8 +462,32 @@ protected override void PopIn()
...
+                if (allFiltered)
+                    continue;
+
+                int columnNumber = nonFilteredColumnCount;
+                Scheduler.AddDelayed(() =>
+                {
+                    var channel = columnAppearSample?.GetChannel();
+                    if (channel == null) return;
+
+                    // don't play sample if column isn't visible, but try to play it at least 5 times (assuming there's at least 5 visible columns in `columnFlow`)
```
Comment (peppy) [inline osu.Game/Overlays/Mods/ModSelectOverlay.cs:474]: This conditional is a tad weird to read. To confirm, the goal is to play the sound even for off-screen columns up to a max of 5 to make the effect sound correct?

Comment (nekodex) [reply osu.Game/Overlays/Mods/ModSelectOverlay.cs:474]: yes

**Comment by peppy** [comment]
Actually after changing the button sound, the balance seems fine. It feels less bad without the button sound hiding the first few columns' sounds.

File: osu.Game/Overlays/Mods/ModSelectOverlay.cs (hunk 1 above)
Comment (peppy) [reply osu.Game/Overlays/Mods/ModSelectOverlay.cs:474]: Hmm, the only discrepancy is that it will play 6 times with the code you have.

**Review by peppy — APPROVED** [review]

**Pull request merged**